ambientweather2sqlite/units_mapping.py

```
@@ -0,0 +1,60 @@
+    "mmhg": "millimeter_Hg",
+    "degC": "celsius",
+    "degF": "fahrenheit",
+    "lux": "lux",
+    "w/m2": "watt / meter ** 2",
+    "fc": "lumen / foot ** 2",
+    "ug/m3": "microgram / meter ** 3",
+    "%": "percent",
+    "°": "angular_degree",
+}
+
+def units_for_columns(labels: dict[str, str], units: dict[Units, str]) -> tuple[dict[str, str], dict[str, str]]:
```

Ruff (0.11.9)

Line too long (112 > 88)

(E501)